vscode/src/jsonrpc/agent-protocol.ts

```diff
@@ -525,6 +594,17 @@ export interface ClientCapabilities {
     // convenient for clients that forward the string directly to an underlying
     // webview container.
     webviewMessages?: 'object-encoded' | 'string-encoded' | undefined | null
+    // Whether the client supports the VSCode WebView API. If 'agentic', uses
+    // AgentWebViewPanel which just delegates bidirectional postMessage over
+    // the Agent protocol. If 'native', implements a larger subset of the VSCode
+    // WebView API and expects the client to run web content in the webview.
+    // Defaults to 'agentic'.
+    webview?: 'agentic' | 'native'
+    // If webview === 'native', describes how the client has configured webview resources.
+    // cspSource is passed to the extension as the Webview cspSource property.
+    // webviewBundleServingPrefix is prepended to resource paths under 'dist' in
+    // asWebviewUri (note, multiple prefixes are not yet implemented.)
+    webviewNativeConfig?: { cspSource: string; webviewBundleServingPrefix: string }
```

Comment:
```suggestion
    webview?: 'agentic' | 'native' | undefined | null
    // If webview === 'native', describes how the client has configured webview resources.
    // cspSource is passed to the extension as the Webview cspSource property.
    // webviewBundleServingPrefix is prepended to resource paths under 'dist' in
    // asWebviewUri (note, multiple prefixes are not yet implemented.)
    webviewNativeConfig?: { cspSource: string; webviewBundleServingPrefix: string } | undefined | null
```

I think these are causing the tests to fail?